test/unit/core/action-creator.ts

```diff
@@ -139,24 +139,29 @@ suite('ActionCreator', ({ expect, spy, stub }) => {
 
     describe('addRefinement()', () => {
       it('should update a value refinement when two parameters are provided', () => {
-        const updateSearch = actions.updateSearch = spy();
         const navigationId = 'book';
         const value = 'a';
 
-        actions.addRefinement(navigationId, value);
+        const result = actions.addRefinement(navigationId, value);
 
-        expect(updateSearch).to.be.calledWith({ navigationId, value });
+        expect(result.type).to.be.eq('UPDATE_SEARCH');
+        expect(result.payload).to.be.eql({ navigationId, value });
```

Comment:
instead, mock the `updateSearch` action and return a mock that you can compare here, we don't want this to break when the functionality of `updateSearch` changes